packages/blockchain-wallet-v4-frontend/src/data/modules/coinify/sagas.js

```diff
@@ -539,6 +562,76 @@ export default ({ coreSagas, networks }) => {
     }
   }
 
+  const recurringCheckoutInitialized = function * () {
+    const initialValues = { frequency: 'weekly', duration: null }
+    yield put(initialize('coinifyRecurringCheckout', initialValues))
+  }
+
+  const handleRecurringFormChange = function * (action) {
+    try {
+      const form = path(['meta', 'form'], action)
+      if (!equals(form, 'coinifyRecurringCheckout')) return
```

Comment:
name of the form can be put to constant, in model.js file ( in the sagas, selectors, ... bundle)